python/tank/authentication/defaults_manager.py

```diff
@@ -22,10 +22,60 @@ class DefaultsManager(object):
     on disk and the system maintains a concept of a current user and a current
     host.
 
+    The defaults manager will also look in multiple locations for a file called ``config.ini``
+    in order to provide appropriate defaults:
+
+        - The SGTK_CONFIG_LOCATION environment variable,
+        - The SGTK_DESKTOP_CONFIG_LOCATION environment variable.
+        - The ~/Library/Application Support/Shotgun/config.ini file
+        - The ~/Library/Caches/Shotgun/desktop/config/config.ini file
+        - One of the fallback locations specified in the :meth:``~DefaultsManager.__init__``
+
+    .. code-block:: ini
+        # Login related settings
+        #
+        [Login]
+
+        # If specified, the username text input on the login screen will be populated
+        # with this value when logging into the Shotgun Desktop for the very first time.
+        # Defaults to the user's OS login.
+        #
+        default_login=$USERNAME
```

Comment:
I'm not sure about this. Maybe we should harmonize these values with what we have in shotgun.yml?
login, host, http_proxy and app_store_http_proxy. We can always provide fallback for default_login and default_site for backwards compatibility in the old Desktop file.